src/components/tx/security/shared/SecurityWarnings/index.tsx

```diff
@@ -0,0 +1,163 @@
+        sx={{ bgcolor: ({ palette }) => palette[severityProps.color].background }}
+        icon={
+          <SvgIcon
+            component={AlertIcon}
+            inheritViewBox
+            color={severityProps.color}
+            sx={{
+              '& path': {
+                fill: ({ palette }) => palette[severityProps.color].main,
+              },
+            }}
+          />
+        }
+      >
+        {severity !== SecuritySeverity.NONE && <Typography variant="h5">{pluralizedLabel}</Typography>}
+        <Box display="flex" flexDirection="column" gap={2}>
+          {warnings.map((warning) => (
+            <Typography key={warning} variant="body2">
+              {warning}
+            </Typography>
+          ))}
+        </Box>
+      </Alert>
+    </>
+  )
+}
+
+export const SecurityWarning = ({
+  severity,
+  isLoading,
+  error,
+  children,
+  needsConfirmation,
+  isConfirmed,
+  setIsConfirmed,
+}: {
+  severity: SecuritySeverity | undefined
+  isLoading: boolean
+  error: Error | undefined
+  children: ReactElement
+  needsConfirmation: boolean
+  isConfirmed: boolean
+  setIsConfirmed: (value: boolean) => void
+}) => {
+  const severityProps = severity !== undefined ? mapSeverityComponentProps[severity] : undefined
+
+  return (
+    <Box className={css.wrapperBox}>
+      <Accordion className={css.verdictBox}>
+        <AccordionSummary sx={{ mb: 0 }} expandIcon={<ExpandMoreIcon />}>
+          <Grid container direction="row" justifyContent="space-between" alignItems="center">
+            <Grid item>
+              <Typography fontWeight={700} variant="subtitle1">
+                Scan for risks
+              </Typography>
+              <Typography
+                color="text.secondary"
+                fontSize="12px"
+                display="flex"
+                flexDirection="row"
+                gap={1}
+                alignItems="center"
+                mt={-1}
+              >
+                Powered by <SvgIcon inheritViewBox sx={{ height: '40px', width: '52px' }} component={RedefineLogo} />
+              </Typography>
+            </Grid>
+            {isLoading ? (
+              <LoadingLabel />
+            ) : error ? (
+              <Typography variant="body2" fontWeight={700} color="error">
+                {error.message}
+              </Typography>
+            ) : (
+              severityProps && (
+                <Typography variant="body2" fontWeight={700} color={`${severityProps.color}.main`}>
+                  {severityProps.label}
+                </Typography>
+              )
+            )}
+          </Grid>
+        </AccordionSummary>
+        <AccordionDetails>{children}</AccordionDetails>
+      </Accordion>
+      {needsConfirmation && (
+        <Box display="flex" flexDirection="row" gap={1} alignItems="center">
+          <Track {...MODALS_EVENTS.ACCEPT_RISK}>
+            <Checkbox checked={isConfirmed} onChange={() => setIsConfirmed(!isConfirmed)} />
```

Comment:
Can we wrap this in a `<FormControlLabel>` similar to the execute checkbox so the whole line is clickable? Otherwise the user has to click directly on the icon which is a bit harder.